frontend/.gitignore

```diff
@@ -10,7 +10,7 @@
 
 # production
 /build
-
+/dist
```

Comment:
Please don't remove the newline. Just add this entry under /build without replacing the newline.